src/lib/db/event-store.ts

```diff
@@ -412,11 +411,7 @@ class EventStore implements IEventStore {
         const rows = await this.db(TABLE)
             .update({ announced: true })
             .where('announced', false)
-            .whereNotNull('announced')
-            .where('created_at', '>', subDays(Date.now(), 1))
-            .returning(EVENT_COLUMNS)
-            .limit(500);
```

Comment:
Note: I've tested that limit does not work with update. There are alternatives but may require using native queries: 
- https://stackoverflow.com/questions/11432155/limiting-postgresql-update-command
- https://www.crunchydata.com/blog/simulating-update-or-delete-with-limit-in-postgres-ctes-to-the-rescue